simpleclient_jetty/src/main/java/io/prometheus/client/jetty/JettyStatisticsCollector.java

```diff
@@ -1,18 +1,17 @@
 package io.prometheus.client.jetty;
```

Comment:
Why is this file in both the jetty and jetty_jdk8 modules? I'd expect it to be in only one of them.